Cinnamenu@json/files/Cinnamenu@json/applet.js

```diff
@@ -272,6 +272,14 @@ CinnamenuApplet.prototype = {
         this.refresh();
       });
 
+      this.panelEditModeChangedId = global.settings.connect('changed::panel-edit-mode', Lang.bind(this, function() {
+         if (!global.settings.get_boolean('panel-edit-mode')) {
+            this.state.orientation = this._orientation;
+            let list_of_enabled_applets = global.settings.get_strv("enabled-applets");
+            this.refresh();
+         }
+      }));
+
```

Comment:
Some nitpicky style things:
 - Use arrow functions over Lang.bind - easier for debugging
 - Connect signal using `this.signals` with signalManager (so it will be disconnected with the other signals on removal)
 - Use camelCase variable names over under_score variable names for consistency

This doesn't work though - you can move a panel without going into panel edit mode, so it won't update in that case. We could move this to [on_orientation_changed](https://github.com/claudiux/cinnamon-spices-applets/blob/d1d3ae27fa0303910ab0be06848bfebf55149986/Cinnamenu%40json/files/Cinnamenu%40json/applet.js#L449) - orientation change is already handled there, if its different than the old orientation, then do the re-render. 

In this case, its actually re-rendering the applet twice - once when going into panel edit mode, and another time when moving the panel. I think it does't propagate on the first re-render - which is why it didn't work in `on_orientation_changed`. I would prefer to look for a more specific rendering selectivity for categories. I appreciate the effort you're putting in here and I agree some customization should be handled here, but it might be easier for me to refactor category building so drag and drop re-ordering works. When that change is implemented, it probably would be cleaner to not keep this option around, since it has a narrower scope for customization than DND category re-ordering.